index.html

```diff
@@ -1177,119 +1177,305 @@ <h2 id="relationship-to-permissions-policy">
         permission=] or by some user agent policy.
       </p>
     </section>
-    <section data-cite="webdriver">
+    <section>
       <h2 id="automation">
         Automated testing
       </h2>
       <p>
         For the purposes of user-agent automation and application testing, this document defines
-        the following <a>extension commands</a> for the [[WebDriver]] specification. It is OPTIONAL
-        for a user agent to support <a>extension commands</a> commands.
+        extensions to the [[WebDriver]] and [[WebDriver-BiDi]] specifications.
+        It is OPTIONAL for a user agent to support them.
       </p>
       <pre class='idl'>
         dictionary PermissionSetParameters {
           required PermissionDescriptor descriptor;
           required PermissionState state;
         };
       </pre>
-      <section>
-        <h3 id="set-permission-command">
-          Set Permission
+      <section data-cite="webdriver-bidi webdriver">
+        <h3 id="automation-webdriver-bidi">
+          Automated testing with [[WebDriver-BiDi]]
         </h3>
-        <table>
-          <tbody>
-            <tr>
-              <th>
-                HTTP Method
-              </th>
-              <th>
-                <a data-lt="extension command URI template">URI Template</a>
-              </th>
-            </tr>
-            <tr>
-              <td>
-                POST
-              </td>
-              <td>
-                /session/{session id}/permissions
-              </td>
-            </tr>
-          </tbody>
-        </table>
         <p>
-          The <dfn class="export" data-dfn-for="extension commands">Set Permission</dfn>
-          <a>extension command</a> simulates user modification of a {{PermissionDescriptor}}'s
-          <a>permission state</a>.
+          This document defines the following [=extension modules=] for the [[WebDriver-BiDi]] specification.
```

Comment:
It seems like this dfn is not exported.

Reply:
There was a bug in webref: https://github.com/w3c/webref/issues/1071

Which is now fixed, so now the `dfn` is exported.

Reply:
Ah, great. Thanks! 🐞 🐛